### DISCUSSION TITLE
OS X Support

### DISCUSSION THREAD

**Discussion by ghostforest** [dfranx/SHADERed#259] [General]
Hey, may I ask for os x support or a guide on how to compile on mac osx?

**Comment by GabZach** [comment]
On my system (Big Sur) it compiled out of the box as advertised in the README.

The only thing that does not work is the right-click - you need to use a 3-button mouse :-(

**Reply by 0xkr4t0s** [reply]
Get the application called middle to work around that I had to use it for using Houdini lol, https://middleclick.app/
